src/qibo/ui/mpldrawer.py

```diff
@@ -0,0 +1,721 @@
+# Some functions in MPLDrawer are from code provided by Rick Muller
+# Simplified Plotting Routines for Quantum Circuits
+# https://github.com/rpmuller/PlotQCircuit
+#
+import json
+from os import path
+
+import matplotlib
+import numpy as np
+
+from qibo import gates
+
+from .FusedGateBarrier import FusedGateEndBarrier, FusedStartGateBarrier
+
+global STYLE
+STYLE = {}
+
+global SYMBOLS
+SYMBOLS = {}
```

Comment:
`global` keyword is only needed in a local scope, here is useless.
```suggestion
STYLE = {}

SYMBOLS = {}
```

Reply:
Moreover, it is almost always a bad idea to use global variables, only use global constants.

You can load them top-level (and never write to them), or pass them as parameters.
```py
from pathlib import Path

UI = Path(__file__).parent
STYLE = json.loads((UI / "styles.json").read_text())
SYMBOLS = json.loads((UI / "symbols.json").read_text())
```